src/app/productConfig/productConfigModal.tpl.html

```diff
@@ -87,38 +87,23 @@ <h4 class="panel-title border-bottom-small mt0 pt-" translate>
             </h4>
             <div class="panel-body pt0">
               <div class="row">
-                <div class="col-sm-4">
+                <div class="col-sm-2">
                   <div class="form-group">
-                    <label translate>Month</label>
+                    <label translate>Day</label>
                     <div class="form-group">
                       <select class="form-control form-control-subtle"
-                              ng-model="$ctrl.itemConfig['start-month']"
-                              ng-change="$ctrl.changeStartMonth($ctrl.itemConfig['start-month'])">
-                        <option value="01" translate>January</option>
-                        <option value="02" translate>February</option>
-                        <option value="03" translate>March</option>
-                        <option value="04" translate>April</option>
-                        <option value="05" translate>May</option>
-                        <option value="06" translate>June</option>
-                        <option value="07" translate>July</option>
-                        <option value="08" translate>August</option>
-                        <option value="09" translate>September</option>
-                        <option value="10" translate>October</option>
-                        <option value="11" translate>November</option>
-                        <option value="12" translate>December</option>
+                              ng-model="$ctrl.itemConfig['recurring-day-of-month']"
+                              ng-options="o as o for o in $ctrl.daysInMonth()"
+                              ng-change="$ctrl.changeStartDay($ctrl.itemConfig['recurring-day-of-month'])">
                       </select>
                     </div>
                   </div>
                 </div>
-                <div class="col-sm-2">
+                <div class="col-sm-4">
                   <div class="form-group">
-                    <label translate>Day</label>
+                    <label>Will start on</label>
```

Comment:
Do you think this should be hidden if there is no day selected yet? It seems a little weird with just a floating heading. Or actually the day is supposed to be defaulted to the nextDrawDate so this won't be an issue.

It could use a `translate` attribute though...